### PR TITLE
AT_055.01 | Pipeline>Check Discard old builds on general configure

### PR DESCRIPTION
**Pull request by AnastasiiaPV**
AT: https://trello.com/c/U70SkaRQ/945-at05501-pipelinecheck-discard-old-builds-on-general-configure
TC: https://trello.com/c/bLAXJ0ly/944-tc055001-pipelinecheck-discard-old-builds-on-general-configure

### CODE REVIEW

File: src/test/java/_PipelineTest.java [hunk 1]
```diff
@@ -434,7 +436,7 @@ public void testCreatePipeline() {
         createPipeline(pipelineName(), Boolean.TRUE);
```
Comment (GitForProjects) [inline src/test/java/_PipelineTest.java:436]: you can use ProjectUtils.createProject - to create new pipeline

File: src/test/java/_PipelineTest.java [hunk 2]
```diff
@@ -848,4 +850,31 @@ public void testCheckScheduledBuildInBuildHistory() {
...
+        getDriver().findElement(By.id("cb2")).click();
+        ProjectUtils.clickSaveButton(getDriver());
+        WebDriverWait wait = new WebDriverWait(getDriver(), 5);
+
+        for (int i = 0; i < 40; i++) {
+            getDriver().findElement(BUILD_NOW_BUTTON).click();
+            Thread.sleep(50);
+        }
+
+        wait.until(ExpectedConditions.visibilityOfElementLocated(
```
Comment (GitForProjects) [inline src/test/java/_PipelineTest.java:867]: we have common method for waits  -  BaseTest.getWait5

**Review by GitForProjects — CHANGES_REQUESTED** [review]

File: src/test/java/_PipelineTest.java (hunk 1 above)
Comment (AnastasiiaPV) [reply src/test/java/_PipelineTest.java:435]: Thank you. Fixed the issues and made a new commit. Please rewiew

File: src/test/java/_PipelineTest.java (hunk 2 above)
Comment (AnastasiiaPV) [reply src/test/java/_PipelineTest.java]: Thank you. Fixed the issues and made a new commit. Please review.

**Pull request closed without merging**